--- a/src/conf.h      2021-11-18 13:09:03.099056356 +0900
+++ b/src/conf.h  2021-11-18 13:10:35.252014907 +0900
@@ -67,11 +67,11 @@
 #define DEFAULT_AUTHSERVSSLAVAILABLE 0
 /** Note:  The path must be prefixed by /, and must be suffixed /.  Put / for the server root.*/
 #define DEFAULT_AUTHSERVPATH "/wifidog/"
-#define DEFAULT_AUTHSERVLOGINPATHFRAGMENT 	"login/?"
-#define DEFAULT_AUTHSERVPORTALPATHFRAGMENT 	"portal/?"
+#define DEFAULT_AUTHSERVLOGINPATHFRAGMENT 	"login?"
+#define DEFAULT_AUTHSERVPORTALPATHFRAGMENT 	"portal?"
 #define DEFAULT_AUTHSERVMSGPATHFRAGMENT 	"gw_message?"
-#define DEFAULT_AUTHSERVPINGPATHFRAGMENT 	"ping/?"
-#define DEFAULT_AUTHSERVAUTHPATHFRAGMENT 	"auth/?"
+#define DEFAULT_AUTHSERVPINGPATHFRAGMENT 	"ping?"
+#define DEFAULT_AUTHSERVAUTHPATHFRAGMENT 	"auth?"
 #define	DEFAULT_AUTHSERVWSPATHFRAGMENT		"ws/"
 /** Note that DEFAULT_AUTHSERVSSLNOPEERVER must be 0 or 1, even if the config file syntax is yes or no */
 #define DEFAULT_AUTHSERVSSLPEERVER 1    /* 0 means: Enable peer verification */
